src/main/kotlin/uk/gov/justice/digital/hmpps/nomisvisitsmappingservice/data/AppointmentMappingDto.kt

```diff
@@ -0,0 +1,33 @@
+package uk.gov.justice.digital.hmpps.nomisvisitsmappingservice.data
+
+import com.fasterxml.jackson.annotation.JsonInclude
+import io.swagger.v3.oas.annotations.media.Schema
+import jakarta.validation.constraints.NotBlank
+import jakarta.validation.constraints.Size
+import uk.gov.justice.digital.hmpps.nomisvisitsmappingservice.jpa.AppointmentMapping
+import java.time.LocalDateTime
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@Schema(description = "NOMIS to Appointment mapping")
+data class AppointmentMappingDto(
+
+  @Schema(description = "Appointment instance id", required = true)
+  val appointmentInstanceId: Long,
```

Comment:
I think we should tend to make these Strings, gives the DPS service flexibility on what sort key they used. Sentencing for instance are using GUUIDS

Reply:
(obviously  the NOMIS one can safely be a long)

Reply:
I went with their proposed database design which does actually show it as a bigint